examples/instrumentation/Wordpress/otel.php.ini

```diff
@@ -0,0 +1,5 @@
+auto_prepend_file=/var/www/otel/autoload.php
+
+[opentelemetry]
+extension=opentelemetry.so
+extension=grpc.so
```

Comment:
I don't see grpc being installed anywhere, and the example is using http/protobuf